homeassistant/components/media_player/gpmdp.py

```diff
@@ -0,0 +1,123 @@
+"""
+Support for Google Play Music Desktop Player.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.gpm_dp/
+"""
+import logging
+
+from homeassistant.components.media_player import (
+    MEDIA_TYPE_MUSIC, SUPPORT_NEXT_TRACK, SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_PAUSE, MediaPlayerDevice)
+from homeassistant.const import (
+    STATE_PLAYING, STATE_PAUSED, STATE_OFF)
+
+_LOGGER = logging.getLogger(__name__)
+REQUIREMENTS = ['websocket-client==0.35.0']
+SUPPORT_GPMDP = SUPPORT_PAUSE | SUPPORT_PREVIOUS_TRACK | SUPPORT_NEXT_TRACK
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the GPMDP platform."""
+    from websocket import create_connection
+    import json
```

Comment:
`json` is a Python built-in package, you can import this at the top of the file.